src/Common/src/System/Net/ContextAwareResult.cs

```diff
@@ -153,7 +153,7 @@ internal ExecutionContext ContextCopy
                 ExecutionContext context = _context;
                 if (context != null)
                 {
-                    return context.CreateCopy();
+                    return context; // No need to copy on CoreCLR; ExecutionContext is immutable
```

Comment:
This code or the same compiled assembly?  If just the code is shared, then we could #ifdef or use a partial class or something.  Shipping with the ExecutionContext.cs shim that's currently in the code isn't a good idea, though, and since the member being used here isn't in the contract, the exact same code can't be used in both the full framework and corefx.